### PR TITLE
0.2.1

### PR DESCRIPTION
**Pull request by hhanesand**
Added public api for switching view controllers
fix selector view not being added sometimes

### CODE REVIEW

**Comment by hhanesand** [comment]
if you're happy with it go ahead and merge (but don't delete :+1: )

File: Pod/Classes/NJHNavigationBarSelectorPageViewController.m [hunk 1]
```diff
@@ -58,6 +59,7 @@ - (void)viewWillAppear:(BOOL)animated {
         for (UIScrollView *view in self.view.subviews) {
             if ([view isKindOfClass:[UIScrollView class]]) {
                 self.pageScrollView = view;
+                self.pageScrollView.backgroundColor = self.scrollViewBackgroundColor;
```
Comment (heumn) [inline Pod/Classes/NJHNavigationBarSelectorPageViewController.m:62]: This seems like a weird way of setting the background color of the container scrollview within the pagecontroller. I thought it was "clearColor" before and thus maybe a more appropriate way of setting the color would be to set the this?

skjermbilde 2015-08-08 kl 11 05 03

I might be misunderstanding what you are doing here :)

**Comment by heumn** [comment]
Very happy :) 

Alt Text

File: Pod/Classes/NJHNavigationBarSelectorPageViewController.m (hunk 1 above)
Comment (hhanesand) [inline Pod/Classes/NJHNavigationBarSelectorPageViewController.m:62]: No you're absolutely right, what I did was an unnecessary complication. Setting the background color of the page controller's view works just as well.

Comment (heumn) [inline Pod/Classes/NJHNavigationBarSelectorPageViewController.m:62]: :+1:

**Pull request merged**